tgdb/playerdetails.php

```diff
@@ -176,6 +176,27 @@
 $tpl->setvar('NOTECOUNT', count($notes));
 $tpl->setvar('NOTETABLEOPEN', (!count($notes) || count($notes) > 5 ? "collapse" : "in"));
 
+$res->free();
+
+$res = $mysqli->query("SELECT timestamp, text, adminckey, type, server FROM".fmttable("messages").$sqlwhere."AND type = 'message' OR type = 'message sent' ORDER BY timestamp DESC;");
```

Comment:
because @Jordie0608 is a meme, the message table doesn't use the same column name as the rest of the tables, so you have to redefine sqlwhere to use `targetckey` and not `ckey`:

```php 
$sqlwhere = " WHERE targetckey = '".esc(keytockey($_GET['ckey'],FALSE))."'";
```

you also left out some spaces and need to use `(` `)` to keep the AND and OR wheres from getting confused:

```php
$res = $mysqli->query("SELECT timestamp, text, adminckey, type, server FROM ".fmttable("messages").$sqlwhere." AND (type = 'message' OR type = 'message sent') ORDER BY timestamp DESC;");
```